### DISCUSSION TITLE
Delist preview releases of PowerShell if they are actually older than the stable release

### DISCUSSION THREAD

**Discussion by hrxn** [PowerShell/PowerShell#20855] [Enhancements]
It's not really an issue, but maybe an enhancement worth considering? I don't know, feel free to comment and correct anything.

The "problem" is that the *Get PowerShell* table listing the links of the release distribution downloads right at the very project homepage is slightly misleading if the current versions of PowerShell are as they are right now, for example.

https://github.com/PowerShell/PowerShell#get-powershell

Because, right now, the situation is as following, regardless of platform:

| PowerShell release type | Version of linked download |
| ------------- | ------------- |
| LTS | `v7.4.0` |
| Stable | `v7.4.0` |
| Preview | `v7.4.0-rc.1` |

I think it's clear now.

Possible solution proposal:
Temporarily remove all entries in the **Downloads (preview)** column, until an actually newer preview release has been published.

Cheers.

**Comment by kilasuit** [comment]
Not really worth the effort to remove and re-add when there will be a newer preview release after not a long amount of time I don't think but marking this for maintainers to review.

**Comment by hrxn** [comment]
Yeah, to be fair, I initially thought that it wouldn't be worth it, anyway.
On the other hand, in four days it will be exactly one month with this "version mishap", so "not a long amount of time" is, well, relative.. ¯\\\_(ツ)\_/¯
